### ISSUE TITLE
[Feature]: Select card to control select and input_select entities

### ISSUE THREAD

**Issue by PaoloTK** [piitaya/lovelace-mushroom#873]
### Requirements

- [X] I have updated Mushroom to the latest available version
- [X] I did a search to see if there is a similar issue or if a pull request is open.

### Is your feature request related to a problem?

Mushroom cannot control select or input_select entities at the moment.

### Describe the solution you'd like

Either a new "Select Card" or the ability to show a select dropdown for entity card when a select or input_select entity is selected.

**Comment by piitaya** [comment]
Thank for your feature request. There are lot of non-supported entity in Mushroom and we can not add support for all the domain because for now : 1 domain = 1 card. And it's a lot a duplicate code.
You can use the more-info popup for now for this.

**Comment by PaoloTK** [comment]
> Thank for your feature request. There are lot of non-supported entity in Mushroom and we can not add support for all the domain because for now : 1 domain = 1 card. And it's a lot a duplicate code. You can use the more-info popup for now for this.

Thank you. Working on the card as we speak, PR requires an open issue on the topic so opened one.

PS: any chance you can take a look at my RGBW light PR so I can move forward with it?

**Comment by piitaya** [comment]
As I work for Nabu Casa now, there will be lot of ongoing change to the HA dashboard so I prefer to slow down the mushroom development to avoid big refactoring 😅

**Comment by PaoloTK** [comment]
> I do not prefer to add additional control for now. The editor is already too cluttered, which is opposite to what I wanted to bring with Mushroom.

I agree that a cluttered editor is bad. However, I think it's pretty trivial to only show relevant options based on the selected entity. For example, selecting an RGBW light entity would hide the "Temperature color control?" toggle. So the number of options in the editor wouldn't increase (it would actually decrease if implemented for every option).

Also, while I agree that showing too many options is not great, we're not talking about adding options for the sake of it. Being able to control the white channel(s) in a light that has them is pretty important and something everyone that has that kind of light needs to be able to do. It's not a niche feature.
